README.md

```diff
@@ -107,6 +107,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 #### React
 
 - [vite-reactts-electron-starter](https://github.com/lTimeless/vite-reactts-electron-starter) - React, TailwindCSS, TypeScript and Electron.
+- [vite-reactts-chakra-starter](https://github.com/Dieman89/vite-reactts-chakra-starter) - React, Typescript, Chakra, Cypress. An included Demo to know where to start with a small written Cypress test to get you started.
```

Comment:
Can you make it shorter? Ideally, it should fit in one line on GitHub. Thanks

Reply:
Hey, sure!

Reply:
> Can you make it shorter? Ideally, it should fit in one line on GitHub. Thanks

Done! :)